pkg/kubernetes/scripts/policy.js

```diff
@@ -190,7 +190,69 @@
 
                 return methods.create([binding], namespace);
             }
+            function removeRoleFromPolicyBinding(policyBinding, project, role, subject) {
```

Comment:
This function seems like a duplicate of removeFromRole() below. I don't think it's necessary if so.